docs/proposals/formatted-output.md

```diff
@@ -116,46 +114,45 @@ oras manifest fetch $REGISTRY/$REPO:$TAG --format json
 
 ```json
 {
-  "reference": " $REGISTRY/$REPO@sha256:8be4c36a29979c72fdd225654498791fb381a7dd8332ade1981274a16220fe1c",
-  "schemaVersion": 2,
-  "mediaType": "application/vnd.oci.image.manifest.v1+json",
-  "digest": "sha256:8be4c36a29979c72fdd225654498791fb381a7dd8332ade1981274a16220fe1c",
-  "artifactType": "application/vnd.unknown.artifact.v1",
-  "content": {
-    "config": {
-      "mediaType": "application/vnd.oci.empty.v1+json",
-      "digest": "sha256:44136fa355b3678a1146ad16f7e8649e94fb4fc21fe77e8310c060f61caaff8a",
-      "size": 2,
-    },
-    "layers": [
-      {
-        "mediaType": "application/vnd.oci.image.layer.v1.tar",
-        "digest": "sha256:6cb759c4296e67e35b0367f3c0f51dfdb776a0c99a45f39d0476e43d82696d65",
-        "size": 14477,
-        "annotations": {
-          "org.opencontainers.image.title": "sbom.spdx"
-        }
-      },
-      {
-        "mediaType": "application/vnd.oci.image.layer.v1.tar",
-        "digest": "sha256:54c0e84503c8790e03afe34bfc05a5ce45c933430cfd9c5f8a99d2c89f1f1b69",
-        "size": 6639,
-        "annotations": {
-          "org.opencontainers.image.title": "scan-test-verify-image.json"
-        }
-      }
-    ],
-  "annotations": {
-    "org.opencontainers.image.created": "2023-12-15T09:41:54Z"
-  }
-},
+	"reference": " $REGISTRY/$REPO@sha256:8be4c36a29979c72fdd225654498791fb381a7dd8332ade1981274a16220fe1c",
+	"schemaVersion": 2,
+	"mediaType": "application/vnd.oci.image.manifest.v1+json",
+	"digest": "sha256:8be4c36a29979c72fdd225654498791fb381a7dd8332ade1981274a16220fe1c",
+	"artifactType": "application/vnd.unknown.artifact.v1",
+	"content": {
+		"config": {
+			"mediaType": "application/vnd.oci.empty.v1+json",
+			"digest": "sha256:44136fa355b3678a1146ad16f7e8649e94fb4fc21fe77e8310c060f61caaff8a",
+			"size": 2
+		},
+		"layers": [{
+				"mediaType": "application/vnd.oci.image.layer.v1.tar",
+				"digest": "sha256:6cb759c4296e67e35b0367f3c0f51dfdb776a0c99a45f39d0476e43d82696d65",
+				"size": 14477,
+				"annotations": {
+					"org.opencontainers.image.title": "sbom.spdx"
+				}
+			},
+			{
+				"mediaType": "application/vnd.oci.image.layer.v1.tar",
+				"digest": "sha256:54c0e84503c8790e03afe34bfc05a5ce45c933430cfd9c5f8a99d2c89f1f1b69",
+				"size": 6639,
+				"annotations": {
+					"org.opencontainers.image.title": "scan-test-verify-image.json"
+				}
+			}
+		],
+		"annotations": {
+			"org.opencontainers.image.created": "2023-12-15T09:41:54Z"
+		}
+	}
 }
 ```
```

Comment:
nit: inconsistent formatting: the indentation are in tabs. should be spaces like other examples.

Reply:
Fixed